localization/sparse_mapping/scripts/rbo.py

```diff
@@ -0,0 +1,322 @@
+"""Rank-biased overlap, a ragged sorted list similarity measure.
```

Comment:
Is this from an open source? We need to check the permissions and give credit if so